app/webpacker/components/Panel/SeniorDelegate/Regions/Subregion.jsx

```diff
@@ -81,13 +130,22 @@ export default function Subregion({ title, groupId }) {
           <Table.Row>
             <Table.HeaderCell>Name</Table.HeaderCell>
             <Table.HeaderCell>Status</Table.HeaderCell>
+            <Table.HeaderCell>Actions</Table.HeaderCell>
           </Table.Row>
         </Table.Header>
         <Table.Body>
           {delegates.map((delegate) => (
             <Table.Row key={delegate.id}>
               <Table.Cell>{delegate.user.name}</Table.Cell>
               <Table.Cell>{I18n.t(`enums.user.role_status.delegate_regions.${delegate.metadata.status}`)}</Table.Cell>
+              <Table.Cell>
+                {canPromote(delegate)
+                  && <Button onClick={() => promoteDelegateAction(delegate)}>Promote</Button>}
+                {canDemote(delegate)
+                  && <Button onClick={() => demoteDelegateAction(delegate)}>Demote</Button>}
+                {!isLead(delegate)
```

Comment:
(sidenote, not blocking: In the medium-term future we should probably have a distinct "pass on the torch" action to transfer leadership. It seems cumbersome to first promote a new member to Leader and technically have two leaders for a short amount of time just so you can demote the ex-leader and end their role)

Reply:
Actually we have that. The management of leaders (or senior delegates or regional delegates) cannot be done from this page or GroupsManager. For that, there will be a page in Board panel. For Senior/Regional Delegates, there is one already (Region Manager), the one for team/committee leaders is under construction, but council leaders is already there but code has to be changed a bit because it's using some async/await which got missed in the review and I came to know it's not good practice in a later PR where it was rejected.